### PR TITLE
Always show hash in `cleos get code` #900

### PR DESCRIPTION
**Pull request by zxcat**

### CODE REVIEW

File: programs/cleos/main.cpp [hunk 1]
```diff
@@ -2776,10 +2776,10 @@ int main( int argc, char** argv ) {
...
          const std::vector<char> abi_v(abi_string.begin(), abi_string.end());
 
-
-         if(!wasm.empty()) {
-            code_hash = fc::sha256::hash(wasm.data(), wasm.size());
-         }
+         fc::sha256 hash;
+         if(wasm.size())
+            hash = fc::sha256::hash(wasm.data(), wasm.size());
+         code_hash = (string)hash;
```
Comment (e-schepachev) [inline programs/cleos/main.cpp]: Don't use the C-style cast. Use the C++ static_/dynamic_/reinterpret cast instead.
As I remember, we always put '{ }' in the if/while/for constructs.

Comment (zxcat) [reply programs/cleos/main.cpp]: it's copy-pasted from eos sources. prefer to keep things not changed if not needed to simplify merge of mainstream changes

Comment (e-schepachev) [reply programs/cleos/main.cpp]: ок.

Comment (zxcat) [reply programs/cleos/main.cpp]: agree, we should refactor it, when cleos become far enough from original

**Pull request merged**